rollup.config.js

```diff
@@ -32,7 +34,13 @@ function createESMConfig(input, output) {
     external,
     plugins: [
       resolve({ extensions }),
-      typescript(),
+      esbuild({
+        minify: process.env.NODE_ENV === 'production',
+        target: 'esnext',
+        jsxFactory: 'React.createElement',
+        jsxFragment: 'React.Fragment',
+        tsconfig: path.resolve('./tsconfig.json'),
+      }),
       babel(getBabelOptions({ node: 8 })),
```

Comment:
Do we still need to run babel? Doesn't esbuild accept target build?

Reply:
You're right, i've removed it

Reply:
Actually, if jotai needs to support ie11, we need to keep it

Reply:
that makes sense.